III-5709 Content Scope (#301)
cultuurnet/apidocs: projects/uitdatabank/docs/content-scope.md

# Content scope

Any public event, activity or location used for leisure purposes can be entered in the UiTdatabank. When entering data, the following publication rules are applicable.

## What may or may not be entered?

<!-- theme: info -->
> 
> In case of violating the rules your event or location will be removed or rejected from the UiTdatabank. If you repeatedly violate these rules, you may be denied access to the UiTdatabank.

### Political events

* ❌ Activities that exclusively express one political opinion (e.g. a speech).
* ✅ Activities involving more than one political party or opinion, or where no political position is taken and the leisure content prevails (e.g. a debate or dinner party).

> ### Religous activities

Reviewer: ```suggestion
### Religious activities
```

* ❌  Activities aimed exclusively at propagating or professing a religion or cult, or aimed at a community of believers or adherents (e.g. weekly Eucharist).
* ✅  Activities that are rooted in religion, but are publicly accessible and where the leisure aspect prevails. Examples: a procession, a performance in a synagogue, or a tour of a mosque.

### Objects and websites
* ❌ A mere object such as a book
* ❌ Promotion for a website
* ✅ Online events

### Hotels, campsites, rooms or restaurants 

> * ❌ Overnight stays, menus (e.g. a valentines menu in a restaurant)

Reviewer: ```suggestion
* ❌ Overnight stays, menus (e.g. a valentines' menu in a restaurant)
```

> * ✅ Leisure events, e.g. a concert or perforamnce in a cafe.

Reviewer: ```suggestion
* ✅ Leisure events, e.g. a concert or performance in a café.
```

### Activities where money and opportunity are central
* ❌ Activities that involve gambling, or that take place at a location where gambling takes place, entail financial risk and risk of addiction
* ✅ Activities such as bingo in the local service center, where these risks are not present.

### Activities that focus on the sale of products or services
* ❌ Activities with a purely commercial purpose or activities that advertise one product or service.
* ✅ Activities where multiple products, services or brands are marketed, and where the leisure experience takes precedence (e.g. market or fair).

### Activities without date and/or location that need to be booked
* ❌ Activities where the date or location depend on mutual agreement or consultation.

> * ✅ An exception to this are bookable school events solely targetted towards schools, and can be booked by a teacher or school for educational purposes.

Reviewer: ```suggestion
* ✅ An exception to this are bookable school events solely targeted towards schools, and can be booked by a teacher or school for educational purposes.
```
